### PR TITLE
Jenkins integration

### PR DESCRIPTION
**Pull request by dolgopolovwork**
Simple Jenkins pipeline.

### CODE REVIEW

**Review by yurivin — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+pipeline {
+
+ options {
+  skipDefaultCheckout()
+  buildDiscarder(logRotator(numToKeepStr: '20'))
+ }
+ agent any
```
Comment (bakhtin) [inline Jenkinsfile]:
```suggestion
 agent { label 'd3-build-agent' }
```

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ stages {
+  stage('Tests') {
+   steps {
+    script {
+     env.WORKSPACE = pwd()
```
Comment (bakhtin) [inline Jenkinsfile]: That is redundant

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+     iC = docker.image("openjdk:8-jdk")
```
Comment (bakhtin) [inline Jenkinsfile]:
```suggestion
     docker.image("openjdk:8-jdk").inside {
```
Do you run any containers during the build (e.g. IrohaContainers)? If not, you do not really need mounting Docker socket inside the container.

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+     iC.inside("-v /var/run/docker.sock:/var/run/docker.sock -v /tmp:/tmp") {
```
Comment (bakhtin) [inline Jenkinsfile]:
```suggestion
     
```

**Review by bakhtin — CHANGES_REQUESTED** [review]
Would also be nice to indent Jenkinsfile with 2 spaces (there is just one now)

File: Jenkinsfile (hunk 3 above)
Comment (dolgopolovwork) [reply Jenkinsfile]: I run Iroha container inside it. It fails if I don't mount the socket.

**Review by bakhtin — APPROVED** [review]

**Review by Mingela — APPROVED** [review]

**Pull request merged**